config/initializers/extends.rb

```diff
@@ -178,7 +178,7 @@ class Extends
     change_task_due_date: 62,
     remove_task_due_date: 63,
     add_task_tag: 64,
-    edit_task_tag: 65,
+    edit_tag: 65,
```

Comment:
Will we have any problems with existing data?

Reply:
It was create, but never used. And in Activity Excel had flag "not implement". So i rename it along with other tag activities.